documentation/modules/vmware-prerequisites.adoc

```diff
@@ -41,6 +41,7 @@ The following minimal set of VMware privileges is required to migrate virtual ma
 2+|`Virtual machine.Interaction` privileges:
 |`Virtual machine.Interaction.Power Off`   |Allows powering off a powered-on virtual machine. This operation powers down the guest operating system.
 |`Virtual machine.Interaction.Power On`  |Allows powering on a powered-off virtual machine and resuming a suspended virtual machine.
+|`Virtual machine.Guest operating system management by VIX API`  |Allows managing a virtual machine by VMware VIX API.
```

Comment:
https://www.vmware.com/products/beta/vmware_server/vix_api/ReferenceGuide/intro.html - just looking at the VMware docs and they use an article... **BUT** i have no issues if you chose to ignore as this is pedantic and it reads well enough without. 

```suggestion
|`Virtual machine.Guest operating system management by VIX API`  |Allows managing a virtual machine by the VMware VIX API.
```